py-polars/tests/unit/operations/test_statistics.py

```diff
@@ -24,6 +24,11 @@ def test_corr() -> None:
     assert_frame_equal(result, expected)
 
 
+def test_corr_nan() -> None:
+    df = pl.DataFrame({"a": [1.0] * 71, "b": [-1.1, 2.2] * 30 + [3.3] * 11})
```

Comment:
I was just about the push a 1-1 similar fix!
But I guess I can only suggestion minor changes now :)
Is it worth simplifying the DataFrame?
```suggestion
    df = pl.DataFrame({"a": [1.0, 1.0], "b": [1.0, 2.0]})
```

Reply:
Yes that's worth it. Thanks.